src/split_single_dyn_dim.cpp

```diff
@@ -67,6 +69,33 @@ has_one_dyn_dim(const std::unordered_map<std::string, shape>& param_shapes)
                                     dds_it->max};
 }
 
+struct find_static_2in_broadcasts
+{
+    // convert 2 input static shape broadcast/multibroadcast into 1 input version
+    auto matcher() const
+    {
+        return match::broadcast(match::nargs(2),
+                                match::arg(0)(match::static_shape()),
+                                match::arg(1)(match::static_shape()));
+    }
+
+    void apply(module& m, const match::matcher_result& mr) const
+    {
+        auto ins          = mr.result;
+        auto out_lens     = ins->get_shape().lens();
+        auto broadcast_op = ins->get_operator();
+        if(broadcast_op.name() == "broadcast")
+        {
+            broadcast_op.from_value({{"out_lens", out_lens}});
```

Comment:
Add case for then branch of If

Reply:
I don't understand

Reply:
I think he means add a test case for this line.